scripts/common-service.Makefile

```diff
@@ -126,7 +126,7 @@ _run-test-ci: _check_venv_active
 		--cov=$(APP_PACKAGE_NAME) \
 		--durations=10 \
 		--asyncio-mode=auto \
-		-m "not travis" \
+		-m "not heavy_load" \
```

Comment:
Can you explain what this does? This will skip all tests regarding file sized in the CI?

Reply:
this allows to skip ANY test that is marked with the marker ```heavy_load```. it is useful for tests that cannot run in the smaller machine like what is available in the CI and which would make the whole testing super slow.